tests/test_pipedrive_start_date.py

```diff
@@ -115,7 +114,10 @@ def test_run(self):
 
                     # Verify the number of records replicated in sync 1 is greater than the number
                     # of records replicated in sync 2 for stream
-                    self.assertGreater(record_count_sync_1, record_count_sync_2)
+                    if stream == "users":
+                        self.assertGreaterEqual(record_count_sync_1, record_count_sync_2)
```

Comment:
How many records we have for users?

Reply:
We have only 1 user record. It is the current pipedrive user.